pkg/crc/input/input.go

```diff
@@ -18,3 +23,15 @@ func PromptUserForYesOrNo(message string, force bool) bool {
 	}
 	return false
 }
+
+func PromptUserForPullSecret(message string) (string, error) {
+	output.OutF(message + ": ")
+	pullsecret, err := terminal.ReadPassword(int(syscall.Stdin))
+	if err != nil {
+		return "", err
+	}
+	if err := validation.ImagePullSecret(string(pullsecret)); err != nil {
+		return "", err
+	}
+	return string(pullsecret), nil
+}
```

Comment:
This function can be generic, used for getting any kind of secret from user, also the validation is already done in https://github.com/code-ready/crc/pull/304/files#diff-3cdf39384e7d570617455a54805a594aR109